Content.Client/Smoking/BurnStateVisualizerSystem.cs

```diff
@@ -0,0 +1,42 @@
+using Robust.Client.GameObjects;
+using Content.Shared.Smoking;
+
+namespace Content.Client.Smoking;
+
+public sealed class BurnStateVisualizerSystem : VisualizerSystem<BurnStateVisualsComponent>
+{
+
+    [Dependency] private readonly IEntityManager _entMan = default!;
```

Comment:
don't need this, EntitySystems have this integrated and have helper methods already

Reply:
removed